AMW_angular/io/package.json

```diff
@@ -59,6 +59,6 @@
     "protractor": "~7.0.0",
     "ts-node": "~8.10.2",
     "tslint": "~6.1.3",
-    "typescript": "~4.0.5"
+    "typescript": "^4.0.7"
```

Comment:
I think this should be change to `~4.1`
Should fix https://github.com/liimaorg/liima/runs/2119744614?check_suite_focus=true#step:7:555

Angular supported version: https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3
node-semver: https://github.com/npm/node-semver#caret-ranges-123-025-004